src/manage_nsfs/nc_master_key_manager.js

```diff
@@ -19,9 +19,36 @@ const TYPE_EXEC = 'executable';
 const ACTIVE_MASTER_KEY = 'active_master_key';
 const exec_key_suffix = 'master_keys';
```

Comment:
didn't we want to add noobaa to the prefix? and this const deserves to be capitalized
```suggestion
const EXEC_KEY_SUFFIX = 'noobaa.master_keys';
```

Reply:
noobaa was the prefix that was mentioned in the script itself

Reply:
ok then, so without the noobaa prefix is ok